std/src/main/scala/algebra/std/byte.scala

```diff
@@ -0,0 +1,58 @@
+package algebra
+package std
+
+import algebra.lattice._
+import algebra.number._
+import algebra.ring._
+import algebra.std.util.StaticMethods
+
+package object byte extends ByteInstances
+
+trait ByteInstances {
+  implicit val byteAlgebra = new ByteAlgebra
+
+  val ByteMinMaxLattice = new Lattice[Byte] {
```

Comment:
@johnynek I chose to create non-implicit lattice instances, under the assumption that there were many possible lattices available (bitwise and/or, min/max, maybe more?).

Maybe this is the wrong strategy?